amundsen_application/api/utils/metadata_utils.py

```diff
@@ -46,10 +47,9 @@ def marshall_table_partial(table_dict: Dict) -> Dict:
 
     TODO - Unify data format returned by search and metadata.
     """
-    schema = PopularTableSchema(strict=True)
-    # TODO: consider migrating to validate() instead of roundtripping
-    table: PopularTable = schema.load(table_dict).data
-    results = schema.dump(table).data
+    schema = PopularTableSchema()
+    table: PopularTable = schema.load(table_dict, unknown=EXCLUDE)
```

Comment:
the lines below insert `key` into the result, however some test cases have `key` as part of table_dict. Since `PopularTable` itself is anyway deprecated, making `load` more lenient made sense.

Reply:
could you help to comment on `PopularTable itself is anyway deprecated` ? Do we plan to replace the popular table section with the full table object?

Reply:
I only based this off the comment in https://github.com/amundsen-io/amundsencommon/blob/master/amundsen_common/models/popular_table.py.